src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import './Header.scss';
+import HomeIcon from '@material-ui/icons/Home';
+import SupervisorAccountIcon from '@material-ui/icons/SupervisorAccount';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import SearchIcon from '@material-ui/icons/Search';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import { NavigationIcon } from '../NavigationIcon/NavigationIcon';
+import { AuthContext } from '../../context/AuthProvider';
+
+export const Header: React.FC = () => {
+  const user = useContext(AuthContext);
+
+  return (
+    <header className="header">
+      <nav className="header__left">
+        <LinkedInIcon className="header__logo" color="primary" />
+        <div className="header__search">
+          <SearchIcon />
+          <input className="header__input" placeholder="Search" type="text" />
+        </div>
+      </nav>
+
+      <nav className="header__right">
```

Comment:
it should be Navigation component which loop throught NavigationItems (use map function)

Reply:
[{path: '', icon" '', title: ''}]

Reply:
create constants folder and file Navigation.ts and store this navigation items there

constants/Navigation.ts

export const navigationLinks = [{path: '', icon" '', title: ''}]

Reply:
ul tag is missing here